Sources/NIO/ServerSocket.swift

```diff
@@ -44,16 +44,22 @@ final class ServerSocket: BaseSocket {
 
     /// Accept a new connection
     ///
+    /// - parameters:
+    ///     - flags: The flags to pass to accept4 on Linux only (eg `SOCK_NONBLOCK`).
     /// - returns: A `Socket` once a new connection was established or `nil` if this `ServerSocket` is in non-blocking mode and there is no new connection that can be accepted when this method is called.
     /// - throws: An `IOError` if the operation failed.
-    func accept() throws -> Socket? {
+    func accept(flags: Int32 = 0) throws -> Socket? {
```

Comment:
Should we use `CInt` ?

Reply:
Yup, prefer `CInt` to `Int32` in this case.